profile.js

```diff
@@ -35,4 +35,64 @@ document.getElementById('menu').addEventListener('click', () => {
   document.getElementById('link1').addEventListener('click', hideMobMenu);
   document.getElementById('link2').addEventListener('click', hideMobMenu);
   document.getElementById('link3').addEventListener('click', hideMobMenu);
-});
\ No newline at end of file
+});
+// || =========== Pop-Up Modal ============ ||
+const arrProjects = [];
+function addProjects() {
+  // Stores Project Infos in arrProject
+  const colTitles = document.getElementsByClassName('project-titles');
+  const colClient = document.getElementsByClassName('client');
+  const colRole = document.getElementsByClassName('type');
+  const colYear = document.getElementsByClassName('year');
+  for (let i = 0; i < colTitles.length; i += 1) {
+    // | creating a new project object an pushing it to arrProject
+    const objProject = {};
+    objProject.name = colTitles[i].textContent;
+    objProject.client = colClient[i].textContent;
+    objProject.role = colRole[i].textContent;
+    objProject.year = colYear[i].textContent;
+    objProject.tags = ['html', 'css', 'javascript', 'github', 'ruby', 'bootstrap'];
+    objProject.snapshot = `images/SnapshootD${i + 1}.jpg`;
+    arrProjects.push(objProject);
+  }
+}
+
+function getProjectDetails(proID) {
+  // Retrieving  Project Info
+  return arrProjects[proID];
+}
+
+function showProjectDetails(proID) {
+  // showing  Project Info
+  const modPopUp = document.getElementById('modal-ui-wrapper');
```

Comment:
- [ ] Please use the correct width for model buttons and make sure it matches the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/169249283-de3fc65a-0055-4444-86a4-cbd7de4135ec.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/169249467-8f19af11-dbb0-43fc-8c39-0e8f047a21df.png)